src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -114,6 +113,7 @@ protected ActionResult execute() throws EntityDoesNotExistException {
             return createAjaxResult(data);
         }
         
+       BackDoorLogic backDoor = new BackDoorLogic();
```

Comment:
CheckStyle will catch this when the rule is up. The indentation is off by one whitespace.